editor/editor_node.cpp

```diff
@@ -472,6 +472,9 @@ void EditorNode::_notification(int p_what) {
 			get_tree()->get_root()->set_as_audio_listener(false);
 			get_tree()->get_root()->set_as_audio_listener_2d(false);
 			get_tree()->set_auto_accept_quit(false);
+#if defined(ANDROID_ENABLED) && defined(TOOLS_ENABLED)
```

Comment:
All of `editor/` is only built with `TOOLS_ENABLED`, so it's redundant to specify it here too.